### PR TITLE
Fix igu graph

### PR DESCRIPTION
**Pull request by jmarrec**
First commit: Strip all output (Cell > All Output > Clear all outputs)

Second is the quick fix, to keep all years inside the dataframe after the stack/unstack you're doing.

Also fixing an unrelated list override

### CODE REVIEW

File: MFA_IGU.ipynb [hunk 1]
```diff
@@ -933,6 +951,15 @@
     "## Consumption of Insulating Glass Units"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "old_index = EU_IGU.index"
```
Comment (jmarrec) [inline MFA_IGU.ipynb:960]: Store old index

File: MFA_IGU.ipynb [hunk 2]
```diff
@@ -952,6 +979,15 @@
     "EU_IGU = EU_IGU.unstack()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "EU_IGU = EU_IGU.reindex(old_index)"
```
Comment (jmarrec) [inline MFA_IGU.ipynb:988]: Reapply old index after stack/unstack, when you "lost" all NaN years

File: MFA_IGU.ipynb [hunk 3]
```diff
@@ -1105,6 +1141,15 @@
...
```
Comment (jmarrec) [inline MFA_IGU.ipynb:1150]: Same, Reapply old index after stack/unstack, when you "lost" all NaN years

File: MFA_IGU.ipynb [hunk 4]
```diff
@@ -1122,7 +1167,7 @@
     "# Number of countries for trend curves:\n",
     "EU_list = [(6, 12), (15, 15), (27, 28)]\n",
     "\n",
-    "for col, data in enumerate(data):\n",
+    "for col, data in enumerate(['Production', 'Estimated consumption']):\n",
```
Comment (jmarrec) [inline MFA_IGU.ipynb:1170]: Here, you're overriding `data` (the list) by using `data` the element iterator

File: MFA_IGU.ipynb (hunk 2 above)
Comment (JeanSouv) [reply MFA_IGU.ipynb:988]: Great, thanks a lot!!

File: MFA_IGU.ipynb (hunk 4 above)
Comment (JeanSouv) [reply MFA_IGU.ipynb:1170]: Well spotted! Thks!

**Pull request merged**